app/lib/apply_api/client.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ApplyApi
+  class Client
+    include HTTParty
+    # TODO: Make it real!
+    base_uri Settings.apply.api_base_url
+    headers "Accept" => "application/json",
+            "Content-Type" => "application/json;odata.metadata=minimal",
+            "Authorization" => -> { "Bearer #{AccessToken.fetch}" },
+            "User-Agent" => "Register for teacher training"
```

Comment:
Great. Could be useful to include the environment here?

Reply:
Good idea! 👍